docs/DeveloperGuide.md

```diff
@@ -313,7 +313,35 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use Case: Scheduling a meeting with a new person**
+
+**MSS**
+
+1. User adds new person to FINDvisor
```

Comment:
Not sure if we should mention the case for add? For editing and deleting, there is no mention of adding a person but I am not sure if the step should be stated in this scenario. What do you all think? @Javiery3889 @Arixeyeion @YuZizhen

Reply:
Perhaps it would be better to start with `1. User adds new person to FINDvisor` for this MSS as the use case is "Scheduling a meeting with **a new person**". On the other hand, if the use case is "Scheduling a meeting with **an existing person**" then the first line can be omitted

Reply:
As Javier mentioned, but it might be better to add the use case for Adding a new person and then we can refer to that use case in line 1.